source/_integrations/zha.markdown

```diff
@@ -612,16 +612,18 @@ The exact method in which these values are reported depends on the Zigbee networ
 
 RSSI (Received Signal Strength Indicator) values are an indicator value of the raw signal strength between two devices. RSSI values are negative numbers in -dBm format (0 to -100 power ratio in decibels of the measured power referenced to one milliwatt). Lower negative values indicate less interference and a good signal. RSSI information is only between the endpoint device and the first hop from that device. As such, it may not necessarily show signal strength to the Zigbee Coordinator but instead could be showing signal strength to the nearest Zigbee Router device.
 
 - Generally, anything -60 and above (meaning -50, -40, etc.) in RSSI should be considered a strong signal (not losing messages).
-- Anything at -80 and below (meaning -85, -90, etc.) should be considered a noise environment and you risk losing messages.
+- Usually, anything at -80 and below (meaning -85, -90, etc.) in RSSI should be considered a noisy environment and you risk losing messages.
 
 LQI (Link Quality Index) values can be hard to interpret for Zigbee. This is because the Zigbee specifications and the (IEEE 802.15.4 specification) do not standardize how to perform LQI measurements. LQI values are shown as positive numbers on a scale. But because the value provided by the Zigbee devices is not measured using unified standards from all device manufacturers and Zigbee stacks, the values can not always be trusted. For example, Zigbee devices based on Silicon Labs EmberZNet stack use positive display numbers for LQI, where higher is better and lower is worse. The Texas Instruments Z-Stack computes LQI for each received packet from the raw “received signal strength index” (RSSI) by linearly scaling it between the minimum and maximum defined RF power levels for the radio that more or less just provides an LQI value that, based on the strength of the received signal.  This can be misleading in case the user has a noisy environment with interference within the same frequency range (as the RSSI value may be shown as increased even though the true link quality decreases). Other manufacturers and Zigbee stacks measure and calculate LQI values in another way.
 
-- In theory, an LQI value of 255 means a zero error rate in theory. In general, a positive high LQI value is better and a lower LQI value is worse. However, depending on your devices, that might not be the reality.
+- In theory, an LQI value of 255 means a zero error rate, meaning it indicates absolute perfect link quality.
+- In general, a positive high LQI value is better and a lower LQI value is worse. However, depending on your devices, that might not always be the reality.
+- Usually, if a device stays at an LQI value of 40+ or above then that link can usually be considered acceptable for normal operational conditions.
```

Comment:
LQI isn't standard so I don't think it's useful to list exact numbers such as this. Furthermore, LQI is often a measure of the *last-hop* link quality, not of the connection between the device and the coordinator. Listing exact numbers leads people to try to mess with their network until the numbers get higher, often times without a problem existing to begin with.

Reply:
Removed numbers.